scene/main/node.cpp

```diff
@@ -1435,6 +1435,18 @@ TypedArray<Node> Node::find_children(const String &p_pattern, const String &p_ty
 	return ret;
 }
 
+void Node::reparent(Node *p_parent, bool p_keep_global_transform) {
+	ERR_FAIL_NULL(p_parent);
+	ERR_FAIL_NULL_MSG(data.parent, "Node needs a parent to be reparented.");
```

Comment:
I don't have a strong opinion here but do we need this check?
I know that the `reparent` method does not really make sense, if the `node` does not have a parent (one can just use `add_child`.
But it is not harmful either.

Reply:
Well, it's not harmful, but the method is meant for reparenting. As you said, one can just use `add_child()`.